src/libsync/creds/oauth.cpp

```diff
@@ -223,8 +223,6 @@ void OAuth::startAuthentication()
     });
     fetchWellKnown();
 
-    openBrowser();
```

Comment:
That one looks fishy without changes to every user of oauth

Reply:
This is a necessary change. Without it, the browser will be opened automatically when moving to the OAuth2 credentials page, which we no longer want to have.

I found the OAuthTest to be broken because of this change, too, but that was an easy fix. Which other users of this method could be affected? We might add a boolean parameter to the function to make this opt-out, too.

Reply:
Can you ensure that loging in works as expected after a log out?

Reply:
Indeed, this is not working any more. So how about the `bool` param?

Reply:
Thats actually better than randomly opening browsers.
But we should probably raise the main windows with a popup asking for reauth.
(Separate pr, I'll open a ticket)

Reply:
Yep. We should also include a button to copy the URL there.